transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystem31Parser.java

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
This whole thing doesn't look right to me. It looks like a copy of 30Parser instead of an extension.